src/ipc/StoreMap.h

```diff
@@ -73,7 +73,9 @@ class StoreMapAnchor
 
 public:
     mutable ReadWriteLock lock; ///< protects slot data below
-    std::atomic<uint8_t> waitingToBeFreed; ///< may be accessed w/o a lock
+    std::atomic<bool> waitingToBeFreed; ///< may be accessed w/o a lock
+    /// true when StoreMap::abortWriting() was called but there were still readers
```

Comment:
I think we can simplify because we do not need to document the free entry state:

```
std::atomic<uint8_t> writerHalted; ///< whether abortWriting() has been called
```

Reply:
Still open.

Reply:
Adding this old problem to the new review: I think we can simplify because we do not need to document the free entry state:

```
std::atomic<uint8_t> writerHalted; ///< whether abortWriting() has been called
```